estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,67 @@
+from datetime import timedelta
+
+from odoo import api, fields, models
+from odoo.exceptions import UserError
+from odoo.tools.translate import _
+
+
+class EstatePropertyOffer(models.Model):
+    _name = 'estate.property.offer'
+    _description = 'An offer for a property'
+    _order = 'price desc'
+
+    price = fields.Float("Price", required=True)
+    status = fields.Selection(string="Status",
+                              selection=[("accepted", "Accepted"),
+                                         ("refused", "Refused")],
+                              copy=False)
+    partner_id = fields.Many2one('res.partner', string="Partner", required=True)
+    property_id = fields.Many2one('estate.property', string="Property", required=True)
+    date_deadline = fields.Date(string="Deadline", compute='_compute_date_deadline', inverse="_inverse_date_deadline")
+    validity = fields.Integer(string="Validity", default=7)
+    property_type_id = fields.Many2one(related='property_id.property_type_id', string="Property Type", store=True)
+
+    _sql_constraints = [
+        ('positive_offer_price', 'CHECK(price > 0)',
+         'The price must be greater than 0.'),
+    ]
+
+    @api.depends('validity')
+    def _compute_date_deadline(self):
+        for record in self:
+            record.date_deadline = record.create_date or fields.Datetime.now() + timedelta(days=record.validity)
+
+    @api.model_create_multi
+    def create(self, vals):
+        recs = super().create(vals)
+        for record in recs:
```

Comment:
I know I told you recs and records yestarday so it's my fault ahaha, could you change it ?
```suggestion
        offers = ....
        for offer in offers:
```